src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -57,6 +67,14 @@ const TicketsPopover: React.FC<{ futureUpcoming: UpcomingEvent[] }> = (
   </Popover>
 );
 
+const MenuPopover = (
+  <Popover id="menu-popover">
```

Comment:
[OOS] Should be fairly easy to create utility function (or extract from previous uses / other PRs) to create `id` values with desired prefix but added random string to it for uses like this one